Index: app/src/main/java/com/example/shinelon/yoursister/yoursister/fragment/PagerFragment.java
===================================================================
--- app/src/main/java/com/example/shinelon/yoursister/yoursister/fragment/PagerFragment.java	(date 1467785581000)
+++ app/src/main/java/com/example/shinelon/yoursister/yoursister/fragment/PagerFragment.java	(date 1467785581000)
@@ -1,25 +0,0 @@
-package com.example.shinelon.yoursister.yoursister.fragment;
-
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-
-import com.qf.administrator.yoursister.R;
-
-/**
- * A simple {@link Fragment} subclass.
- */
-public class PagerFragment extends Fragment{
-
-    public PagerFragment(){
-        // Required empty public constructor
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState){
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_pager, container, false);
-    }
-}
Index: app/src/main/java/Adapter/SearchRecyclerView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/Adapter/SearchRecyclerView.java	(date 1467785581000)
+++ app/src/main/java/Adapter/SearchRecyclerView.java	(revision )
@@ -66,6 +66,7 @@
                     holder.btn.setTextColor(Color.WHITE);
                     holder.btn.setBackgroundColor(Color.RED);
                     pos=getLayoutPosition();*/
+                    notifyItemRemoved(getLayoutPosition());
                     Toast.makeText(context, "position:" + getLayoutPosition(), Toast.LENGTH_SHORT).show();
                 }
             });
Index: app/src/main/java/com/example/shinelon/yoursister/yoursister/MainActivity.java
===================================================================
--- app/src/main/java/com/example/shinelon/yoursister/yoursister/MainActivity.java	(date 1467785581000)
+++ app/src/main/java/com/example/shinelon/yoursister/yoursister/MainActivity.java	(date 1467785581000)
@@ -1,147 +0,0 @@
-package com.example.shinelon.yoursister.yoursister;
-
-import android.content.Intent;
-import android.graphics.drawable.ColorDrawable;
-import android.os.Bundle;
-import android.support.design.widget.TabLayout;
-import android.support.v4.view.ViewPager;
-import android.support.v7.app.AppCompatActivity;
-import android.view.Gravity;
-import android.view.View;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
-import android.widget.PopupWindow;
-import android.widget.RadioButton;
-import android.widget.RadioGroup;
-import android.widget.RelativeLayout;
-import android.widget.Toast;
-
-import com.qf.administrator.yoursister.R;
-import com.qf.administrator.yoursister.SearchActivity;
-import com.qf.administrator.yoursister.fragment.PagerFragment;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import Adapter.ViewPagerAdapter;
-
-public class MainActivity extends AppCompatActivity implements View.OnClickListener{
-
-    private TabLayout tabLayout;
-    private ViewPager viewPager;
-    private List<PagerFragment> fragments;
-    private RadioButton daogouBTN;
-    private RadioButton garageBTN;
-    private RadioButton hudongBTN;
-    private RadioButton ownBTN;
-    private RadioButton centerBTN;
-    private RadioGroup radioGroup;
-    private PopupWindow popupWindow;
-    private RelativeLayout relativelayout;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState){
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-        initData();
-        initView();
-        ViewPagerAdapter adapter = new ViewPagerAdapter(getSupportFragmentManager(), fragments);
-        viewPager.setAdapter(adapter);
-        tabLayout.setupWithViewPager(viewPager);
-        initPopWindow();
-        clickRadioGroup();
-    }
-
-    /**
-     * 点击RadioGroup里面内容的监听器
-     */
-    public void clickRadioGroup(){
-        radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener(){
-            @Override
-            public void onCheckedChanged(RadioGroup group, int checkedId){
-                switch(checkedId){
-                    case R.id.center:
-                        popupWindow.showAtLocation(relativelayout, Gravity.BOTTOM, 200, 160);
-                        break;
-                }
-            }
-        });
-    }
-
-    private void initView(){
-        relativelayout = (RelativeLayout) findViewById(R.id.mainRelative);
-        radioGroup = (RadioGroup) findViewById(R.id.radioGroup);
-        tabLayout = (TabLayout) findViewById(R.id.tablayout);
-        viewPager = (ViewPager) findViewById(R.id.viewPager);
-        daogouBTN = (RadioButton) findViewById(R.id.daogou);
-        garageBTN = (RadioButton) findViewById(R.id.garage);
-        hudongBTN = (RadioButton) findViewById(R.id.hudong);
-        ownBTN = (RadioButton) findViewById(R.id.own);
-        centerBTN = (RadioButton) findViewById(R.id.center);
-    }
-
-    private void initData(){
-        fragments = new ArrayList<>();
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-        fragments.add(new PagerFragment());
-    }
-
-    /**
-     * 跳转到搜索activity
-     *
-     * @param view
-     */
-    public void searchButton(View view){
-        startActivity(new Intent(this, SearchActivity.class));
-    }
-
-    /**
-     * 这里写popwinow的相关控件设置方法
-     */
-    private void initPopWindow(){
-        View layout = getLayoutInflater().inflate(R.layout.popwindow_layout, null);
-        ImageView postImage = (ImageView) layout.findViewById(R.id.postmsg);
-        ImageView shaiImage = (ImageView) layout.findViewById(R.id.shaiimage);
-        ImageView askImage = (ImageView) layout.findViewById(R.id.askques);
-        ImageView findImage = (ImageView) layout.findViewById(R.id.findcar);
-        postImage.setOnClickListener(this);
-        shaiImage.setOnClickListener(this);
-        askImage.setOnClickListener(this);
-        findImage.setOnClickListener(this);
-        popupWindow = new PopupWindow(layout, LinearLayout.LayoutParams.MATCH_PARENT, ViewPager.LayoutParams.WRAP_CONTENT);
-        //要想在点击popwindow之外的时候能关闭pop,必须设置下面三个参数
-        popupWindow.setOutsideTouchable(true);
-        popupWindow.setBackgroundDrawable(new ColorDrawable(0x000000));
-        popupWindow.setFocusable(true);
-    }
-
-    /**
-     * popwindow里面的事件响应
-     *
-     * @param v
-     */
-    @Override
-    public void onClick(View v){
-        switch(v.getId()){
-            case R.id.postmsg:
-                Toast.makeText(this, "发帖", Toast.LENGTH_SHORT).show();
-                break;
-            case R.id.shaiimage:
-                Toast.makeText(this, "晒图", Toast.LENGTH_SHORT).show();
-                break;
-            case R.id.askques:
-                Toast.makeText(this, "求问", Toast.LENGTH_SHORT).show();
-                break;
-            case R.id.findcar:
-                Toast.makeText(this, "找车", Toast.LENGTH_SHORT).show();
-                break;
-        }
-    }
-}
Index: app/src/main/java/com/example/shinelon/yoursister/yoursister/SearchActivity.java
===================================================================
--- app/src/main/java/com/example/shinelon/yoursister/yoursister/SearchActivity.java	(date 1467785581000)
+++ app/src/main/java/com/example/shinelon/yoursister/yoursister/SearchActivity.java	(date 1467785581000)
@@ -1,107 +0,0 @@
-package com.example.shinelon.yoursister.yoursister;
-
-import android.content.Intent;
-import android.content.SharedPreferences;
-import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.RecyclerView;
-import android.support.v7.widget.SearchView;
-import android.support.v7.widget.StaggeredGridLayoutManager;
-import android.util.Log;
-import android.view.View;
-import android.widget.ArrayAdapter;
-import android.widget.AutoCompleteTextView;
-import android.widget.Button;
-import android.widget.EditText;
-import android.widget.ListView;
-import android.widget.Toast;
-
-import com.qf.administrator.yoursister.*;
-
-import java.util.ArrayList;
-import java.util.List;
-
-/**
- * 这里是首页tablayout右边搜索按钮进来之后出现的界面
- *
- */
-public class SearchActivity extends AppCompatActivity{
-    private EditText search_edittext;
-    private Button search_button;
-    private RecyclerView search_recyclerView;
-    private ListView search_listView;
-    private List<String> titleList;
-    @Override
-    protected void onCreate(Bundle savedInstanceState){
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_search);
-        search_edittext= (EditText) findViewById(R.id.search_edittext);
-        search_button= (Button) findViewById(R.id.search_button);
-        search_recyclerView= (RecyclerView) findViewById(R.id.search_recyclerview);
-        search_listView= (ListView) findViewById(R.id.search_listview);
-        Log.e("init: ", "1");
-        init();
-    }
-
-
-    private void init() {
-        titleList = new ArrayList<>();
-        titleList.add("首页");
-        titleList.add("车评");
-        titleList.add("视频");
-        titleList.add("长策");
-        titleList.add("策划");
-        titleList.add("观点");
-        titleList.add("图片");
-        titleList.add("资讯");
-        titleList.add("经典");
-        Log.e("init: ", "123123");
-        // 获取搜索记录文件内容
-        SharedPreferences sp = getSharedPreferences("search_history", MODE_PRIVATE);
-        String history = sp.getString("history", "暂时没有搜索记录");
-        Toast.makeText(SearchActivity.this, "1", Toast.LENGTH_SHORT).show();
-        // 用逗号分割内容返回数组
-        String[] history_arr=history.split(",");
-        //新建适配器，适配器数据为搜索历史文件内容
-        ArrayAdapter arr_adapter=new ArrayAdapter<String>(this,
-                android.R.layout.simple_dropdown_item_1line, history_arr);
-        //保留前20条
-        if(history_arr.length>20){
-            String[] newArrays=new String[20];
-            // 实现数组之间的复制
-            System.arraycopy(history_arr,0,newArrays,0,20);
-            arr_adapter=new ArrayAdapter<String>(this,
-                    android.R.layout.simple_dropdown_item_1line, history_arr);
-        }
-        search_listView.setAdapter(arr_adapter);
-
-        // 设置监听事件，点击搜索写入搜索词
-        search_button.setOnClickListener(new Button.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                // TODO Auto-generated method stub
-                save();
-                startActivity(new Intent(SearchActivity.this, com.qf.administrator.yoursister.MainActivity.class));
-                Toast.makeText(SearchActivity.this, "123", Toast.LENGTH_SHORT).show();
-            }
-
-        });
-
-    }
-
-    private void save() {
-        String text=search_edittext.getText().toString();
-        SharedPreferences mysp = getSharedPreferences("search_history", MODE_PRIVATE);
-        String old_text=mysp.getString("history","暂时没有搜索记录");
-
-        // 利用StringBuilder.append新增内容，逗号便于读取内容时用逗号拆分开
-        StringBuffer stringBuffer=new StringBuffer(old_text);
-        stringBuffer.append(text+",");
-        // 判断搜索内容是否已经存在于历史文件，已存在则不重复添加
-       if(!old_text.contains(text+",")){
-            SharedPreferences.Editor myeditor=mysp.edit();
-            myeditor.putString("history",stringBuffer.toString());
-           myeditor.commit();
-       }
-    }
-}
Index: app/src/main/java/com/qf/administrator/yoursister/SearchActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/qf/administrator/yoursister/SearchActivity.java	(date 1467785581000)
+++ app/src/main/java/com/qf/administrator/yoursister/SearchActivity.java	(revision )
@@ -77,7 +77,6 @@
             @Override
             public void onClick(String text) {
                 search_recyclerview_button.setText(text);
-                adapter.notifyDataSetChanged();
                 if(flag){
                     search_recyclerView.setVisibility(View.GONE);
                     flag=false;
